app/models/alert_date.py

```diff
@@ -24,6 +24,18 @@ def date_as_lang(self):
         dt = self._local_datetime
         return f'{dt:%A} {dt.day} {dt:%B} {dt:%Y}'
 
+    @property
+    def as_url(self):
+        """
+        * non-zero padded day
+        * lower case month abbrevation
+        * full year
+
+        3-jun-2021
```

Comment:
Should check if there’s precedent on GOV.UK for using `jun` or `june` in URLs

Reply:
@karlchillmaid please do confirm if a change is needed to the formatting of the month

Reply:
Just spotted that we have a url that goes against this convention in production
https://www.gov.uk/alerts/29-june-2021

Meaning either this PR needs to change to not use the month abbreviation or we need to change the url to `jun` and setup a redirect for that alert

Reply:
From the document we wrote when we agreed the URL structure in July:

> The month will always be abbreviated to 3 letters.
> ...
> We discussed whether to format the month as a word (in keeping with GOV.UK style) or a number. We felt that a mixture of words and numbers would make the URL friendlier.
> ...
> We also discussed whether we should use the full month or a 3 letter abbreviation. For example ‘september’ or ‘sep’. The GOV.UK style guide says: ‘when space is an issue - in tables or publication titles, for example - you can use truncated months: Jan, Feb’.

Therefore it should be `jun` instead of `june` – it’s always tricky with the shorter months, but we should always consider the worst case (September) as our baseline.

Reply:
Karl has confirmed we are going for abbreviation still

I have now deployed a redirect for https://www.gov.uk/alerts/29-june-2021 to https://www.gov.uk/alerts/29-jun-2021 so I think we can now move forward on this.